docs/usage.md

```diff
@@ -81,15 +81,25 @@ cd <repo root dir>/docs/examples
 make clean
 ```
 
-### Log your build
+### Generate a compilation database
+
+There are different ways to generate a 
+[compilation database](https://clang.llvm.org/docs/JSONCompilationDatabase.html).
+This compilation database is an input of the next analysis step.
+
+You can check [LLVM documentation](https://clang.llvm.org/docs/JSONCompilationDatabase.html)
+for the different options on how to generate this file.
+
+Alternatively, you can run `CodeChecker log` command to generate a compilation database.
```

Comment:
It is not a true equal alternative. For example, if code is generated during the build process (such as in the case of LLVM), then the CMake-generated compilation database will not be usable as an input if you don't also execute a build prior to analysis.